src/s1reader/s1_orbit.py

```diff
@@ -277,24 +337,24 @@ def get_orbit_file_from_dir(zip_path: str, orbit_dir: str, auto_download: bool =
         return
 
     # Attempt auto download
-    orbit_file = download_orbit(zip_path, orbit_dir)
+    orbit_file = retrieve_orbit_file(zip_path, orbit_dir, concat_resorb)
     return orbit_file
 
 
-def get_orbit_file_from_list(zip_path: str, orbit_file_list: list) -> str:
+def get_orbit_file_from_list(zip_path: str, orbit_file_list: list, concat_resorb: bool=False) -> str | list | None:
     '''Get orbit file for a given S-1 swath from a list of files
 
-    Parameters:
-    -----------
+    Parameters
+    ----------
     zip_path : string
         Path to Sentinel1 SAFE zip file. Base names required to adhere to the
         format described here:
         https://sentinel.esa.int/web/sentinel/user-guides/sentinel-1-sar/naming-conventions
     orbit_file_list : list
         List of the orbit files that exists in the system.
 
-    Returns:
-    --------
+    Returns
+    -------
     orbit_file : str
         Path to the orbit file, or an empty string if no orbit file was found.
     '''
```

Comment:
```
t_swath_start_stop[0] = t_swath_start_stop[0] - margin_start_time
```
similar to heresh's naming comming, maybe we can unpack this pair of times and name them more specifically

Reply:
I would like to keep the original code that represents the time window as a list. So I'm making the list's name more clear that it's about the search window, not the sensing start / stop.

```python
if all([t_orbit_start < t < t_orbit_stop for t in t_search_window]):
```